scripts/las2grain.py

```diff
@@ -197,16 +197,32 @@ def grain_pipeline(cal_las, shp_fp_rfl,imu_data, known_rfl,
 def aart_1064(rfl_grid):
 
 
-
     '''
     Solves Optical Grain Size for AART at 1064 nm wavelength. 
     Assumes pixel is 100% snow and that lidar is pure backscatter.
     g=0.75 and b=1.6 are assumed
-    This is a polynomial approximation that behaves quite well when compared to AART.
-    R2 = 1.0 ;mean absolute error = 0.19 μm. 
+    This is an exact solution using algebra of Kokhanovsky et al. (2021).  
 
     '''
-
-    grain_grid = 18008 * (rfl_grid**6)  - 82840 * (rfl_grid**5)  + 161006 * (rfl_grid**4) - 171899 * (rfl_grid**3)  + 109726 * (rfl_grid**2)- 41764 * (rfl_grid**1)  + 7774.1  
-
-    return grain_grid
+    # Constants
+    lam = 1064*1e-9
```

Comment:
Minor style comment and I use all CAPS for constants. Better highlights differences between variable names in the code